sample/build.gradle.kts

```diff
@@ -54,6 +75,16 @@ kotlin {
         jvmMain.dependencies {
             implementation(compose.desktop.currentOs)
         }
+
+        val jsWasmMain by creating {
+            dependsOn(commonMain)
+        }
+        val jsMain by getting {
+            dependsOn(jsWasmMain)
+        }
+        val wasmJsMain by getting {
+            dependsOn(jsWasmMain)
+        }
```

Comment:
I don't we need to define those source sets. I tried to run the app without it and it worked. I think every target depends automatically on commonMain.
I downloaded the current template from https://kmp.jetbrains.com/. It also doesn't define those source sets.

Reply:
Sometime `wasmJs` and `Js` shared code (javascript browser code) so we create this common js target 

the sample does not have a common code right now so it will pass 

but you can see this usage in this [jetbrains example](https://github.com/JetBrains/compose-multiplatform/blob/1b2669f59d8f9a5cda1a26388a51e8c6793981c8/examples/imageviewer/shared/build.gradle.kts#L64)

I would recommend keeping it, but i can remove no issue up to you

Reply:
I see. That makes sense, let's keep it.